lib/ansible/plugins/action/__init__.py

```diff
@@ -346,8 +346,11 @@ def _execute_module(self, module_name=None, module_args=None, tmp=None, task_var
         # set no log in the module arguments, if required
         if self._play_context.no_log:
             module_args['_ansible_no_log'] = True
+            module_display_args = "(no_log enabled, args censored)"
+        else:
+            module_display_args = module_args
 
-        self._display.vvv("Module: %s, args: %s" % (module_name, module_args))
+        self._display.vvv("Module: %s, args: %s" % (module_name, module_display_args))
```

Comment:
well, the problem only exists if we try to report on the arguments BEFORE executing the module, which we really should not need, that entry was original a debug entry that was added because we rewrote the whole engine, we should not need this anymore.